src/batch/src/error.rs

```diff
@@ -59,6 +84,49 @@ pub enum BatchError {
 
     #[error("Task aborted: {0}")]
     Aborted(String),
+
+    #[error(transparent)]
+    PbFieldNotFound(#[from] PbFieldNotFound),
+
+    #[error(transparent)]
+    RpcError(
+        #[from]
+        #[backtrace]
+        RpcError,
+    ),
+
+    #[error("Connector error: {0}")]
+    Connector(
+        #[source]
+        #[backtrace]
+        BoxedError,
+    ),
+
+    #[error(transparent)]
+    Shared(
+        #[from]
+        #[backtrace]
+        Arc<Self>,
+    ),
```

Comment:
~~What is this `Shared` for? Perhaps need some comments~~

Okay, it means `Arc` 😄